xml/System.Security.Claims/Claim.xml

```diff
@@ -488,9 +497,9 @@ if (null != principal)
         <ReturnType>System.Byte[]</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>Returns <see cref="T:System.Byte" />.</value>
-        <remarks>To be added.</remarks>
+        <summary>Contains any additional data provided by a derived type. Typically set when calling <see cref="M:System.Security.Claims.ClaimsIdentity.WriteTo(System.IO.BinaryWriter,System.Byte[])" />.</summary>
```

Comment:
I would make the second sentence a complete sentence with a subject, which I'd put in a Remarks section.